.golangci.yml

```diff
@@ -12,11 +12,109 @@ run:
     - ^api
     - ^proto
     - ^.git
-issues:
-  # Show only new issues: if there are unstaged changes or untracked files,
-  # only those changes are analyzed, else only changes in HEAD~ are analyzed.
-  # It's a super-useful option for integration of golangci-lint into existing large codebase.
-  # It's not practical to fix all existing issues at the moment of integration:
-  # much better don't allow issues in new code.
-  new-from-rev: HEAD~
-  fix: true
+linters-settings:
+  govet:
+    fieldalignment: 0
+  revive:
+    severity: error
+    confidence: 0.8
+    rules:
```

Comment:
Is it possible not to include enabled rules? If new rules are added later I would like them to be enabled by default and only some of them explicitly disabled.

Reply:
Yes it is. Fixed

Reply:
Please fix PR description also :-)